liberty-managed/src/test/java/io/openliberty/arquillian/managed/WLPResourceTestCase.java

```diff
@@ -0,0 +1,37 @@
+package io.openliberty.arquillian.managed;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import javax.annotation.Resource;
+import org.junit.Assert;
+
+@RunWith(Arquillian.class)
+public class WLPResourceTestCase {
+
+    @Deployment
+    public static JavaArchive createDeployment() {
+        return ShrinkWrap.create(JavaArchive.class)
+                .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml");
+    }
+
+    @Resource(lookup = "env/foo")
+    String foo;
+
+    @Test
+    public void serverXMLJNDIEntryShouldNotBeNull(){
+        Assert.assertNotNull(foo);
```

Comment:
when we are checking the value of foo here and in the other test with `System.getenv` we mind as well test for the value `bar` since we know what the exact value will be